Configuration/StandardSequences/python/Eras.py

```diff
@@ -66,7 +66,8 @@ def __init__(self):
                            'run2_miniAOD_devel', 'run2_nanoAOD_102Xv1',
                            'hcalHardcodeConditions', 'hcalSkipPacker',
                            'run2_HLTconditions_2016','run2_HLTconditions_2017','run2_HLTconditions_2018',
-                           'bParking']
+                           'bParking',
+                           'run3_RPC']
```

Comment:
@dilsonjd mostly aesthetics that can be adjusted separately, but it would make sense to my mind to have the the run3 modifiers in sequence

Reply:
ok. I will add it to the line 57.